### ISSUE TITLE
Classes of ambiguity

### ISSUE THREAD

**Issue by KnowledgeGarden** [jliphard/CancerBase#10]
- Nothing happening on the front page except "clickable" buttons which have an affect on dots on the map. No obvious way to "browse" what's going on  "inside" the portal. Could impact the "stickiness" of the portal.

- FAQ hints at the implementation with instructions to professionals to use PubNub, which is likely not free. Seems important, in the context of personal medical information, to know what the business model is: are your users also your products?

- I registered, but still cannot sign in.

- Signin on the front page is not a link or button. Just one login by facebook button strongly implies this is a facebook-only website-- not until you use the menu do you learn there are other options.

-

**Comment by jliphard** [comment]
GREAT - keep the comments coming!

**Comment by KnowledgeGarden** [comment]
A guess is that you are using the node authentication packages associated with express.js; those can be somewhat tricky to tune. I strongly vote for making this platform open source, say, right here in this repo. I say that because of the opportunity for collaboration, code sharing, helping each other, and much more. In that context, I would like to add that the license chosen does matter.  I'd like to say more: this site's database is ripe for topic mapping; that's my game. My own platform TQPortalKS in my repo is driven by a topic map. This site could be as well.
In relation to PubNub, I'd like to learn what use case it satisfies.

**Comment by KnowledgeGarden** [comment]
Another region of ambiguity, albeit perhaps only for me. I signed up at PatientsLikeMe and CureTogether many moons back. At each of those websites, there was no ambiguity in terms of what to expect; that along the dimension that the website told me nothing on the front page that made me wonder what's inside.  Here, the site says "Let’s connect. The more people we reach, the more people we help. Share your experience with your family and friends. Spread the word!"   How will this site share my experiences with my family and friends? Will it email them something?  The more people we help. How will it help?
Keep in mind that the front page is a very nice map with a growing number of dots.  My own UX expectations are that I can click on any one of those dots and see the data on that individual. That, it turns out, is a failed expectation.

**Comment by jliphard** [comment]
Hi Jack - 
> My own UX expectations are that I can click on any one of those dots and see the data on that individual. That, it turns out, is a failed expectation.
> 

That is a great idea and easy to implement (at least while there are still relatively few people on the map). We could enable that. 

1/ One thing that we need to solve soon has to to do with how to display the data.

Assume many many people enter Detroit as their location. What then? A large dot whose radius scales with number of people? Change the color? Give each dot a random offset to scatter them around the specified location? 

2/ Also agree WRT ambiguity. There is a mismatch between the front page, and the actual content as it stands now on the “SEE DATA” page.

Question - what’s the single most useful thing we could do for the patient? We should add that to the “SEE DATA” page, and then reduce the ambiguity on the front page so there is not ambiguity.

It would be super if you could help us to make specific changes - i.e. what is the problem of the site now, and how specifically do we fix it!

I’ll split off the display question into a separate issue report for ease of tracking. 

Can you think about #2 and give us a recommendation?

Jan

**Comment by regrounding** [comment]
_My own UX expectations are that I can click on any one of those dots and see the data on that individual. That, it turns out, is a failed expectation._

I completely understand this expectation - and many levels share it. I want to enagage with the map on some level. However, the ability to click on a dot and get an individual's data is a gross violation of privacy. My cousin with chordoma is literally 1 in a million. She lives in NY, but if she lived in Milwaukee we identify her almost immediately.

**Comment by KnowledgeGarden** [comment]
So, that raises a new question in regards to the overall platform architecture; ideally, this should, and maybe will be raised as a new issue, but here's the gist:  **Strong Authentication** is crucial to **trust** at any patient-centered epistemic community, but individual identity is more like branding, less like actual individual precise identity.  Experiences with MMOs suggest that a lot of good can happen when people are visible (identified) by avatars, personas which do not carry the same label as their personal names.  If that's the case, then outsiders identifying individuals becomes more problematic.

But, your point about clicking on a map is well taken, regardless of how the site handles people's name strings. Still, sites like patientslikeme do, indeed, present and even advertise (as if some kind of badge of courage or something)  individuals by their real names.  If asked, I would prefer to live behind an avatar on such sites, even in cases where people eventually figure out who I am.

This is a rather complex thread. In any case, the map still, like it or not, invites clicking on dots.

**Comment by KnowledgeGarden** [comment]
Not done with ambiguities yet.
Consider what happens when I save an update to my survey. I see this:
"Thank you! Now have a look at the data others have contributed at the SEE DATA page."
So, I visit the data page. What I am really looking for (obviously not the map) is covered with this statement:
"We will also add other tabs so you can begin to dig into the data. "
That's why I would visit the data page.  How can I help to get the platform there?
